examples/run_language_modeling.py

```diff
@@ -94,6 +102,10 @@ class DataTrainingArguments:
         default=False,
         metadata={"help": "Whether distinct lines of text in the dataset are to be handled as distinct sequences."},
     )
+    lazy_loading: bool = field(
```

Comment:
Can we call this 

```suggestion
    lazy_line_by_line: bool = field(
```
instead?